pkg/nsx/services/securitypolicy/firewall.go

```diff
@@ -28,59 +24,17 @@ const (
 	ProjectTagCount             int = 1
 )
 
-type SecurityPolicyService struct {
+type Service struct {
+	common.Service
```

Comment:
Client client -> what is purpose of this property?  Can it be moved to common.Service as well?